src/js/routes/result/Result.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import BackButton from '../../components/BackButton.jsx';
+
+const Result = (props) => (
```

Comment:
Kind of resolved. Left the Result page component there for now, just to wait and see. Corrected its syntax. It's ready to stay or go, no matter what.